spec.html

```diff
@@ -10440,6 +10440,27 @@ <h1>Static Semantics: CodePointAt ( _string_, _position_ )</h1>
       </emu-alg>
     </emu-clause>
 
+    <emu-clause id="sec-stringindexof" aoid="StringIndexOf">
+      <h1>Static Semantics: StringIndexOf ( _string_, _searchValue_, _fromIndex_ )</h1>
+      <p>The abstract operation StringIndexOf takes arguments _string_ (a String), _searchValue_ (a String), and _fromIndex_ (a non-negative integer). It performs the following steps when called:</p>
+      <emu-alg>
+        1. Assert: Type(_string_) is String.
+        1. Assert: Type(_searchValue_) is String.
+        1. Assert: ! IsNonNegativeInteger(_fromIndex_) is *true*.
+        1. Let _len_ be the length of _string_.
+        1. If _searchValue_ is the empty String, and _fromIndex_ &le; _len_, return _fromIndex_.
+        1. Let _searchLen_ be the length of _searchValue_.
+        1. If there exists any integer _k_ such that _fromIndex_ &le; _k_ &le; _len_ - _searchLen_ and for all nonnegative integers _j_ less than _searchLen_, the code unit at index _k_ + _j_ within _string_ is the same as the code unit at index _j_ within _searchValue_, let _pos_ be the smallest (closest to *-&infin;*) such integer. Otherwise, let _pos_ be -1.
+        1. Return _pos_.
+      </emu-alg>
+      <emu-note>
+        <p>If _searchValue_ is empty and _fromIndex_ is less than or equal to the length of _string_, this algorithm returns _fromIndex_. An empty _searchValue_ is effectively found at every position within a string, including after the last code unit.</p>
+      </emu-note>
+      <emu-note>
+        <p>This algorithm always returns -1 if _fromIndex_ > the length of _string_.</p>
```

Comment:
`&gt;` instead of >?

Reply:
Happy to make the change, but note that it doesn't really matter. `foo > bar` is valid HTML (and valid ecmarkup).